build/ferretdb/eval-dev.Dockerfile

```diff
@@ -1,6 +1,6 @@
 # syntax=docker/dockerfile:1
 
-# for evaluation (formerly "all-in-one") releases (`ferretdb-eval` image)
+# for evaluation (formerly "all-in-one" or "ferretdb-eval") releases (`ferretdb-eval-dev` image)
```

Comment:
Well, "ferretdb-eval" is not a former name – we going to introduce it in the next PR

Reply:
It actually wasn't called that, too